privacy-configuration/config4_reference.json

```diff
@@ -0,0 +1,60 @@
+{
+    "readme": "This configuration has only tracking link features",
```

Comment:
let's update this description

Reply:
Are there guidelines for this description? It appeared that it mostly described the config's contents.

Reply:
Ah, sorry, not sure why I thought it's copied over from other config and not updated, you clearly updated it.